custom_components/ocpp/manifest.json

```diff
@@ -4,14 +4,16 @@
   "documentation": "https://github.com/lbbrhzn/ocpp/blob/main/README.md",
   "iot_class": "local_push",
   "issue_tracker": "https://github.com/lbbrhzn/ocpp/issues",
-  "version": "0.4.3",
+  "version": "0.4.4",
   "config_flow": true,
-  "after_dependencies": ["persistent_notification"],
+  "after_dependencies": [
+    "persistent_notification"
+  ],
   "codeowners": [
     "@lbbrhzn"
   ],
   "requirements": [
-    "ocpp>=0.8",
+    "ocpp==0.11.0",
     "websockets>=10.0"
```

Comment:
Is it worth setting this to ==10.1 so that upstream changes don't affect us or can be tested first?